apple-nav/src/App.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react";
+import { Route } from "react-router-dom";
+import NavData from "./nav_data/NavData";
+import NavWrapper from "./components/NavWrapper";
+// import Nav from "./components/Nav";
+// import SubNav from "./components/SubNav";
+import "./App.css";
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      NavData: []
```

Comment:
I'd recommend sticking to lowercase conventions unless it's a component -- it won't hurt anything, but just for consistency's sake and to avoid confusion. same goes for the data itself when you pull it in (although you can't always control what the data is being exported as)